### ISSUE TITLE
Add formatting of crash reports (currently crashes the logger)

### ISSUE THREAD

**Issue by peerst** [grisp/dgram_logger#6]
When we not only get progress reports but a crash dump the formatting functions can't deal with the fact that the passed list is not of the format `[{K1, V1}, {K2, V2}, ...]` but contains a nested list like:

```
(<8964.1345.0>) call dgram_logger:report_values([{proc_lib,crash},
 [{initial_call,{simple_ctrl,init,['Argument__1']}},
  {pid,<8964.1345.0>},
  {registered_name,simple_ctrl},
  {error_info,
      {error,
          {onewire,nothing_present},
          [{onewire_ds18b20,select_device,1,
               [{file,
                    "/Users/peer/projects/twitch/gsolctrl/_build/default/lib/grisp/src/onewire_ds18b20.erl"},
                {line,45}]},
           {onewire_ds18b20,'-convert/2-fun-0-',2,
               [{file,
                    "/Users/peer/projects/twitch/gsolctrl/_build/default/lib/grisp/src/onewire_ds18b20.erl"},
                {line,32}]},
           {grisp_onewire,handle_call,3,
               [{file,
                    "/Users/peer/projects/twitch/gsolctrl/_build/default/lib/grisp/src/grisp_onewire.erl"},
                {line,133}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,661}]},
           {gen_server,handle_msg,6,[{file,"gen_server.erl"},{line,690}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}},
  {ancestors,[gsolctrl_sup,<8964.106.0>]},
  {message_queue_len,1},
  {messages,[measure]},
  {links,[<8964.107.0>,<8964.110.0>]},
  {dictionary,[]},
  {trap_exit,false},
  {status,running},
  {heap_size,6772},
  {stack_size,27},
  {reductions,284290}],
 []]) ({dgram_logger,send,4})

```

**Comment by peerst** [comment]
Kind of fixed that, by adding a catch all case.  So its no longer crashing but making it nicer by taking apart the legacy data structure we get via the "report" key and format it

**Comment by peerst** [comment]
Closing this issue since we fixed the bug, made a new issue #7 to improve the formatting
